common/controllers/framework/framework-step.js

```diff
@@ -2,10 +2,11 @@ const { isEmpty, fromPairs, snakeCase } = require('lodash')
 
 const fieldHelpers = require('../../helpers/field')
 const frameworksHelpers = require('../../helpers/frameworks')
+const YouthRiskAssessmentService = require('../../services/youth-risk-assessment')
 const FormWizardController = require('../form-wizard')
 const services = {
   'person-escort-record': require('../../services/person-escort-record'),
-  'youth-risk-assessment': require('../../services/youth-risk-assessment'),
+  'youth-risk-assessment': new YouthRiskAssessmentService(),
```

Comment:
Ooooh, we probably want to move where this is instantiated to be within the controller somehow, so it's instantiated with the req. Perhaps services can be set in a constructor method in the controller, which first calls super() and then sets this.services